src/data/resolvers/motions.ts

```diff
@@ -425,11 +426,37 @@ export const motionsResolvers = ({
           },
         });
         if (data?.userReputation) {
-          const reward = await votingReputationClient.getVoterReward(
-            bigNumberify(motionId),
-            bigNumberify(data.userReputation),
-          );
-          return reward.toString();
+          let reward = bigNumberify(0);
+          let minReward = bigNumberify(0);
+          let maxReward = bigNumberify(0);
+          if (state === NetworkMotionState.Submit) {
+            try {
+              const [
+                minUserReward,
+                maxUserReward,
+              ] = await votingReputationClient.getVoterRewardRange(
+                bigNumberify(motionId),
+                bigNumberify(data.userReputation),
+                createAddress(userAddress),
+              );
+              minReward = minUserReward;
+              maxReward = maxUserReward;
+            } catch (error) {
+              /* Not ideal, but when we not logged in useLoggedInUser is returning wrong userAddress
+               When we pass wrong address to getVoterRewardRange method it returning error.
+              */
+            }
```

Comment:
This should be handled on the contracts side. We should be able to call this with any valid wallet address and get some value back, not just a user that voted. Otherwise, this can be used to discover who voted and who didn't.

Please tell Alex about this.